contracts/src/core/BatcherPaymentService.sol

```diff
@@ -109,38 +155,42 @@ contract BatcherPaymentService is
 
         // call alignedLayerServiceManager
         // with value to fund the task's response
-        AlignedLayerServiceManager.createNewTask{value: feeForAggregator}(
+        alignedLayerServiceManager.createNewTask{value: feeForAggregator}(
             batchMerkleRoot,
             batchDataPointer
         );
 
-        payable(BatcherWallet).transfer(
+        payable(batcherWallet).transfer(
             (feePerProof * signaturesQty) - feeForAggregator
         );
     }
 
     function unlock() external whenNotPaused {
-        require(
-            UserData[msg.sender].balance > 0,
-            "User has no funds to unlock"
-        );
+        if (userData[msg.sender].balance <= 0) {
+            revert UserHasNoFundsToUnlock(msg.sender);
+        }
 
-        UserData[msg.sender].unlockBlock = block.number + UNLOCK_BLOCK_COUNT;
+        userData[msg.sender].unlockBlock = block.number + UNLOCK_BLOCK_COUNT;
     }
 
     function lock() external whenNotPaused {
-        require(UserData[msg.sender].balance > 0, "User has no funds to lock");
-        UserData[msg.sender].unlockBlock = 0;
+        if (userData[msg.sender].balance <= 0) {
+            revert UserHasNoFundsToLock(msg.sender);
+        }
+        userData[msg.sender].unlockBlock = 0;
     }
 
     function withdraw(uint256 amount) external whenNotPaused {
-        UserInfo storage user_data = UserData[msg.sender];
-        require(user_data.balance >= amount, "Payer has insufficient balance");
+        UserInfo storage user_data = userData[msg.sender];
```

Comment:
user_data should probably be senderData

Reply:
Solved in [a0fdcd2](https://github.com/yetanotherco/aligned_layer/pull/843/commits/a0fdcd20075b73bc1ac1cdc0a66ae28d1eb89b2a)